src/_sass/base/_base.scss

```
@@ -265,3 +265,17 @@ blockquote {
     text-align: center;
   }
 }
+
+summary {
+  margin-bottom: 1rem;
+}
+
+ol + img, ul + img, ol + p, ul + p, p + p + img {
+  margin-bottom: 1rem;
+}
+
+.figure-caption {
+  font-size: .875rem;
```

You can use `bootstrap.$font-size-sm` which is `.875` of the base font size. That way it's easy to track and change in the future.

```suggestion
  font-size: bootstrap.$font-size-sm;
```